### ISSUE TITLE
SealedSecret seems broken

### ISSUE THREAD

**Issue by nabadger** [bitnami-labs/kube-libsonnet#14]
A couple of issues I'm finding:

- It references `.spec.data` instead of `.spec.encryptedData`.
- It assumes we're only passing in a single value (which it attempts to base64decode)

Something like this works for me:

```
# Create example secret and extra only the encrypted portion into `data.json`
kubectl create secret generic mysecret --dry-run --from-literal=foo=bar -o json >mysecret.json
kubeseal < mysecret.json | jq -r .spec.encryptedData > data.json
```
```yaml
// Define SealedSecret (kube.libsonnet)
SealedSecret(name): $._Object('bitnami.com/v1alpha1', 'SealedSecret', name) {
},
```
```yaml
// Create sealed-secret (calling code)
my_secret: kube.SealedSecret("mysecret") { 
    spec: {
      encryptedData: kubecfg.parseJson(importstr "./data.json"),
    },
  },
```

**Comment by iremmats** [comment]
Was scanning through this too. It should be encryptedData.

**Comment by jjo** [comment]
Fixed by #36.
